packages/s3-credentials-endpoint/index.js

```diff
@@ -92,6 +95,112 @@ async function displayS3CredentialInstructions(_req, res) {
   res.send(compiled(process.env));
 }
 
+/**
+ * If DISABLE_S3_CREDENTIALS is not "true", returns undefined, otherwise, send a
+ * boom.ServerUnavailable to the caller, Exiting the request.
+ *
+ * @param {Object} res - express request object
+ * @returns {undefined} - when DISABLE_S3_CREDENTIALS is not 'true'
+ */
+function ensureEndpointEnabled(res) {
```

Comment:
This and the one below should probably be unit tested.

Reply:
Is this not sufficient?  https://github.com/nasa/cumulus/blob/mhs/CUMULUS-2354/allow-cmr-acl/packages/s3-credentials-endpoint/tests/test-index.js#L345

That was the test before I extracted this function.

Reply:
That's fine